guides/common/modules/con_converting-a-host-to-rhel.adoc

```diff
@@ -0,0 +1,75 @@
+[id="converting-a-host-to-rhel_{context}"]
+= Converting a Host to {RHEL}
+
+You can convert {RHEL} derivative distributions into a supportable {RHEL} on a host while retaining installed applications and configurations.
+{Project} provides the Convert2RHEL utility to simplify the conversion process.
+
+The Convert2RHEL utility in {Project} consists of an Ansible role and an Ansible playbook.
+You use the Ansible role to generate conversion data on {ProjectServer}, that is, repositories, products, activation keys, and host groups.
+Then you perform the actual conversion on the host using the Ansible playbook.
+
+You can convert:
+
+* CentOS Linux 7 to {RHEL} 7
+* Oracle Linux 7 to {RHEL} 7
+* CentOS Linux 8 to {RHEL} 8
+* Oracle Linux 8 to {RHEL} 8
+
+You cannot convert a version 7 system into {RHEL} 8, or a version 8 system into {RHEL} 7.
```

Comment:
Is it really necessary to explain what it _cannot_ do?

Can you convert Rocky Linux 8 to RHEL8? AlmaLinux?

Reply:
The situation is this: The listed conversion paths are officially supported by RH. There are other possible conversion paths, which are described in an article that is referenced in Additional resources below.
I wonder if we should mention those links for non-Satellite as well, because users eventually need to access RH resources anyway, such as for the subscription manifest.

Reply:
Maybe reword to "If you want to convert EL7 to EL8, see LEAPP" which I think is documented for Foreman Server? Otherwise, leave as is; I might create a follow-up PR to extend it slightly.

Reply:
The CLI utility can convert other systems, but the Ansible role and playbook provides only the listed conversions. (Verified with developers.)

Reply:
Seriously, I wouldn't complicate this and go just with the listed conversion paths as supported by the tool.